src/components/EditPlayer/EditPlayer.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from "react";
+import { Link } from "react-router-dom";
+
+import "./editPlayer.css";
+import Button from "../Button";
+import PlayerForm from "../PlayerForm/PlayerForm";
+import { GameContext } from "../../contexts/gameContext";
+
+const EditPlayer = props => {
+  const { id } = props.match.params;
+  const game = useContext(GameContext);
+  const [players, setPlayers] = game.usePlayers;
+  const [playerToEditId] = useState(id);
+
+  const handlePlayerUpdate = player => {
+    const newData = [...players];
+
+    if (player.id) {
+      const playerIndex = players.findIndex(p => {
+        return p.id === player.id;
+      });
+      newData[playerIndex] = player;
+
+      setPlayers(newData);
+      props.history.push("/configure");
+    }
+  };
+
+  return (
+    <>
```

Comment:
nit: turn the fragment into `<div className="edit-player">` to set the BEM block for the heading below